api/db/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Apr 01 12:34:20 EDT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
I did this to make the `db/build.gradle` project importable in IntelliJ. I'm not sure this is the right way to do it or if I still need it. [#dev-productivity thread](https://pmi-engteam.slack.com/archives/CN4AX0K9V/p1585759155001600).